.pre-commit-config.yaml

```diff
@@ -1,5 +1,21 @@
 ---
 repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks.git
+  rev: v2.4.0
+  hooks:
+  - id: end-of-file-fixer
+  - id: trailing-whitespace
+    exclude: |
+      (?x)(
+        ^test/(with-skip-tag-id|unicode).yml|
```

Comment:
@ssbarnea is it possible to move this caret outside of the braces and not repeat it on every line? It's really not necessary here.

Reply:
In fact, the outer braces aren't necessary w/o it

Reply:
Here's a better example: https://pre-commit.com/#regular-expressions